api/spec/serializers/spree/v2/storefront/menu_item_serializer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Spree::V2::Storefront::MenuItemSerializer do
+  subject { described_class.new(menu_item) }
+
+  let(:menu_item) { create(:menu_item) }
+
+  it { expect(subject.serializable_hash).to be_kind_of(Hash) }
+
+  it do
+    expect(subject.serializable_hash).to eq(
```

Comment:
@tomurb same here, different tests for different menu item types - https://github.com/spree/spree/blob/master/core/app/models/spree/menu_item.rb#L21